tests/integration/test_alerts.py

```diff
@@ -47,6 +49,23 @@
         ALERT_ADVANCED_QUERY_COMMAND,
     ],
 )
-def test_alert_command_returns_success_return_code(command, command_runner):
-    return_code, response = command_runner(command)
-    assert return_code == 0
+def test_alert_command_returns_success_return_code(
+    runner, integration_test_profile, command
+):
+    assert_test(runner, integration_test_profile, command)
+
+
+@pytest.mark.integration
+@pytest.mark.parametrize(
+    "command,protocol",
+    [
+        ("alerts send-to localhost:5140 -p TCP -b {}".format(begin_date_str), "TCP"),
+        ("alerts send-to localhost:5140 -p UDP -b {}".format(begin_date_str), "UDP"),
+    ],
+)
+def test_alerts_send_to(runner, integration_test_profile, command, protocol):
```

Comment:
If we are going to use click's runner here, we might as well do it for all the other tests.

Reply:
Done.